torch/csrc/sparse/CusparseLtKernels.cpp

```diff
@@ -0,0 +1,316 @@
+#include <ATen/cuda/CUDAContext.h>
+#include <ATen/cuda/CUDADataType.h>
+#include <c10/core/ScalarType.h>
+#include <c10/util/Half.h>
+#include <cusparse.h>
+#include <torch/csrc/sparse/cusparseLt.h>
+#include <torch/custom_class.h>
+#include <torch/torch.h>
+
+#define CHECK_CUDA(func)                                    \
```

Comment:
You should properly guard this on cuda being available instead of redefining macros we already have.